Lib/filecmp.py

```diff
@@ -138,6 +138,8 @@ def __init__(self, a, b, ignore=None, hide=None, *, shallow=True): # Initialize
         self.shallow = shallow
 
     def phase0(self): # Compare everything except common subdirectories
+        # Do not protect os.listdir() against OSError or ValueError.
+        # See https://github.com/python/cpython/issues/122400.
```

Comment:
I wanted to put this comment so that we do not forget about why there is no protection here but there is one in `phase2`. Do you prefer me removing it?

Reply:
There are tests for this.